### PR TITLE
Fixes issue #464 at the specification level

### PR DESCRIPTION
**Pull request by sandoche2k**

### CODE REVIEW

File: docs/specifications/tests/Address-TP/address01.md [hunk 1]
```diff
@@ -30,59 +30,10 @@ The domain name to be tested.
 2. Each IP address has to be checked against the blocks listed in the three
    tables below.
```
Comment (matsduf) [inline docs/specifications/tests/Address-TP/address01.md]: The tables are not there anymore in the updated version. Please update, @sandoche2k.

File: docs/specifications/tests/Address-TP/address01.md [hunk 2]
```diff
@@ -92,7 +43,7 @@ forbidden blocks, the test case succeeds.
 
 ### Special procedural requirements
 
-None.
+Should automatically fetches the registry mentioned above when need for testing
```
Comment (matsduf) [inline docs/specifications/tests/Address-TP/address01.md]: The language needs some correction and clarification. Suggestion: "The registries listed in bullet 3 must be fetched prior to testing."

**Comment by matsduf** [comment]
@sandoche2k, can you address the issue so that we can merge the change?

**Comment by sandoche2k** [comment]
@matsduf thanks for the input. I have updated accordingly.

**Pull request merged**